### ISSUE TITLE
Minimal Tetra compatibility

### ISSUE THREAD

**Issue by moonfather1** [infernalexp/Mining-Master#25]
### Describe the Addition

 - No not about using gems in a workstation -
 (that's a bit advanced and requires a bunch of json files; also i don't feel the need)

Issue:
MM tools do not qualify as tools when creating something at tetra worktable. 
Example: I want a new pickaxe to have a spruce wood handle; for that I need a tier 1 axe in my inventory or hanging on a rack nearby; a simple stone axe is enough in my inventory to click the button and give my tool a wooden handle. But - none of the MM axes qualify.

**Comment by Dariensg** [comment]
Could you clarify what version this is for? I've just looked at 1.16.5 and the Mining Master tools seem to be backed by the ToolType that Tetra requires to detect that, since they all extend the Vanilla tool items.

**Comment by moonfather1** [comment]
MM 3.0.4, Tetra 3.20

**Comment by Noobulus** [comment]
If memory serves, Tetra uses replacement files (such as [this one](https://github.com/mickelus/tetra/blob/1.18/src/main/resources/data/tetra/replacements/material/diamond.json) for diamond tools) to determine tool level for use in the workbench. Is the 1.18 version of Mining Master still being maintained for me to write a PR for this?

**Comment by Dariensg** [comment]
Yes! 1.18 is still being maintained and a PR would be much appreciated for this!
